src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/DoubleSlider.cs

```diff
@@ -7,18 +7,39 @@ namespace CoreNodeModelsWpf.NodeViewCustomizations
 {
     public class DoubleSliderNodeViewCustomization : INodeViewCustomization<DoubleSlider>
     {
+        private NodeView viewNode;
+        private DoubleSlider sliderModel;
+        private DynamoSlider sliderDynamo;
         public void CustomizeView(DoubleSlider model, NodeView nodeView)
         {
-            var slider = new DynamoSlider(model, nodeView)
+            viewNode = nodeView;
+            sliderModel = model;
+            viewNode.MouseEnter += ViewNode_MouseEnter;
+        }
+
+        private void ViewNode_MouseEnter(object sender, System.Windows.Input.MouseEventArgs e)
+        {
+            ConstructView();
```

Comment:
so this is a little worrying - I think we'd want to add this to some base class to help authors not need to do this themselves, and we'll call ConstructView or whatever for them?

Reply:
something like that yes, we'll need a better mechanism to handle this different than mouse enter event